ui/components/issuessection/issuessection.go

```diff
@@ -222,10 +233,14 @@ func (m *Model) FetchSectionRows() tea.Cmd {
 	if m == nil {
 		return nil
 	}
-	m.Issues = nil
-	m.Table.ResetCurrItem()
-	m.Table.Rows = nil
-	m.IsLoading = true
+
+	if !m.Cursor.HasNextPage {
```

Comment:
I think I added the reset logic here because of when the user refreshes the section. So please check that refresh still works